includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -1,6 +1,11 @@
 <?php
 
 class DesignSystemApiController extends WikiaApiController {
+	const PARAM_PRODUCT = 'product';
+	const PARAM_ID = 'id';
+	const PARAM_LANG = 'lang';
+	const PRODUCT_WIKIS = 'wikis';
+
 	private $data = [
 		"logo" => [
 			"header" => [
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "header" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)